src/toil/jobStores/aws/jobStore.py

```diff
@@ -743,9 +743,13 @@ def bucket_creation_pending(error):
                         if create:
                             logger.debug("Creating bucket '%s'.", bucket_name)
                             location = region_to_bucket_location(self.region)
```

Comment:
We probably don't need [`region_to_bucket_location()`](https://github.com/DataBiosphere/toil/blob/releases/5.4.x/src/toil/jobStores/aws/utils.py#L439) anymore, or location will never be `"us-east-1"`?